tests/test_routes.py

```diff
@@ -173,13 +174,17 @@ def test_update_recommendation(self):
         self.assertEqual(updated_recommendation[PRODUCT_ID], 100)
         self.assertEqual(updated_recommendation[PRODUCT_NAME], "Hat")
 
-        response2 = self.client.put(
-            f"{BASE_URL}/99999",  #non-existing id
-            json=new_rec,
-            content_type=CONTENT_TYPE_JSON,
-        )
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
+    def test_delete_recommendation(self):
```

Comment:
Probably leave other's testing case still?

Reply:
Ok, I will fix it